src/pages/Decks.js

```diff
@@ -0,0 +1,319 @@
+import Filter from '../components/Filter';
+import StyledButton from '../components/StyledButton';
+import Card from '../components/Card';
+import Navigation from '../components/navigations/Navigation';
+import CreateDeckModal from '../components/modals/CreateDeckModal';
+import useCategory from '../hooks/useCategory';
+import useDifficulty from '../hooks/useDifficulty';
+import useCardDecks from '../hooks/useCardDecks';
+import DeleteModal from '../components/modals/DeleteModal';
+import useCards from '../hooks/useCards';
+
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { saveToLocal, loadFromLocal } from '../utils/localStorage';
+import styled from 'styled-components';
+
+export default function Decks({
+  allCategories,
+  onDeleteConfirm,
+  onKeepConfirm,
+  showModal,
+  onCountRights,
+  onCountWrongs,
+  onTrashClick,
+  onPinClick,
+}) {
+  const [showCreateDeckModal, setShowCreateDeckModal] = useState(false);
+  const [cardDeck, setCardDeck] = useState(loadFromLocal('cardDeck') ?? []);
+
+  const navigate = useNavigate();
+
+  const {
+    difficulty,
+    setDifficulty,
+    handleDifficultyCards,
+    easyActive,
+    mediumActive,
+    difficultActive,
+  } = useDifficulty();
+  const { category, handleChange, handleResetFilter, setCategory } =
+    useCategory();
+  const { cards } = useCards();
+  const { doneCards, setDoneCards } = useCardDecks();
+
+  let currentCard = cardDeck[0];
+
+  console.log(cards);
+  console.log(cardDeck);
+
+  useEffect(() => {
+    saveToLocal('difficulty', difficulty);
+    saveToLocal('easyActive', easyActive);
+    saveToLocal('mediumActive', mediumActive);
+    saveToLocal('difficultActive', difficultActive);
+    saveToLocal('cardDeck', cardDeck);
+    saveToLocal('doneCards', doneCards);
+  }, [
+    cardDeck,
+    difficulty,
+    doneCards,
+    easyActive,
+    mediumActive,
+    difficultActive,
+  ]);
+
+  if (cardDeck.length === 0 && doneCards < 5)
+    return (
+      <FlexWrapper>
+        {showCreateDeckModal && (
+          <CreateDeckModal
+            onChangeFilterClick={handleChangeFilterClick}
+            onCreateCardClick={handleCreateCardClick}
```

Comment:
Maybe you can try it like this and delete the function below
```suggestion
            onCreateCardClick={()=> navigate('/create-card')}
```